### PR TITLE
Add new website cosmicscansID, komikbree, rrscans, moonscans

### PR DESCRIPTION
**Pull request by KazielZoe**
Add new website cosmicscans.id, komikbree.xyz, rrscans.my.id, moonscans.xyz

### CODE REVIEW

File: src/web/mjs/connectors/CosmicScansID.mjs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import WordPressMangastream from './templates/WordPressMangastream.mjs';
+
+export default class CosmicScans extends WordPressMangastream {
```
Comment (MikeZeDev) [inline src/web/mjs/connectors/CosmicScansID.mjs]: * Class must match filename and be unique

File: src/web/mjs/connectors/KomikBree.mjs [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import WordPressMadara from './templates/WordPressMadara.mjs';
+
+export default class Aiinscan extends WordPressMadara {
```
Comment (MikeZeDev) [inline src/web/mjs/connectors/KomikBree.mjs]: * Class must match filename and be unique

File: src/web/mjs/connectors/MoonScans.mjs [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import WordPressMangastream from './templates/WordPressMangastream.mjs';
+
+export default class CosmicScans extends WordPressMangastream {
```
Comment (MikeZeDev) [inline src/web/mjs/connectors/MoonScans.mjs]: * Class must match filename and be unique

File: src/web/mjs/connectors/RRScans.mjs [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import WordPressMangastream from './templates/WordPressMangastream.mjs';
+
+export default class CosmicScans extends WordPressMangastream {
```
Comment (MikeZeDev) [inline src/web/mjs/connectors/RRScans.mjs]: * Class must match filename and be unique

**Review by MikeZeDev — CHANGES_REQUESTED** [review]
* Class names fixes required
* Please provide 4 icons for the 4 websites.

**Comment by MikeZeDev** [comment]
Icons files are way to big. Ideally they should be png, < 16kb and dimension max 96x96 px

**Comment by MikeZeDev** [comment]
Cosmics Scans pictures are behind Cloudflare , you should add refer and origin headers otherwise it doesnt work.

```
        this.requestOptions.headers.set('x-referer', this.url);
        this.requestOptions.headers.set('x-origin', this.url);
```

image
image

**Comment by MikeZeDev** [comment]
Many komikbree mangas have 'Bahasa indonesia' in name, You may want to get rid of that
Same for RRscans

Also RR scans chapters randomly doesnt work because ads are messing with us (Girl und panzer)
Not sure something can be done about that. No problem if you cant fix btw.

image

**Comment by 27136773** [comment]
@KazielZoe 
No response since more than `6 weeks` after the last review feedback, is this still worked on?

Would be sad if we had to abandon the changes because of inactivity...

**Pull request closed without merging**